inductiva/molecules/scenarios/protein_solvation/README.md

```diff
@@ -1,17 +1,17 @@
 # Protein Solvation Scenario
 
-This scenario models the dynamics of a protein solvated in water inside a virtual cubic box. The system is modelled with the motion equations for each individual atoms according to the prescription of a molecular force field approximation.
+This scenario models the dynamics of a protein solvated in water inside a cubic box. The system is modelled with the motion equations for each individual atoms according to the prescription of a molecular force field approximation.
```

Comment:
would it be relevant to point to some online resource that has more info on the exact things were computing? like the formula, etc.